lib/flop_phoenix/pagination.ex

```diff
@@ -21,6 +21,7 @@ defmodule Flop.Phoenix.Pagination do
       ],
       next_link_content: "Next",
       page_links: :all,
+      pagination_list_item_attrs: [class: "pagination-list-item"],
```

Comment:
I don't think we need a default class here.

```suggestion
      pagination_list_item_attrs: [],
```